src/App.js

```diff
@@ -31,7 +34,9 @@ function App(props) {
   const { t, i18n } = useTranslation();
 
   const [chosenLanguage, setChosenLanguage] = useState('en');
-  const [isModalOpen, setModalOpen] = useState(!window.localStorage.getItem(UPLOAD_SETTINGS_KEY));
+  const [isModalOpen, setModalOpen] = useState(
+    false && !window.localStorage.getItem(UPLOAD_SETTINGS_KEY)
```

Comment:
Doesn't this just evaluate to always `false`?

Reply:
Argh, that is a leftover after a broken develop.opencast.org. I'll remove this.